test/CMakeLists.txt

```diff
@@ -25,34 +27,6 @@ else()
     message(STATUS "Tests build type is ${CMAKE_BUILD_TYPE}")
 endif()
 
-include(CheckCXXCompilerFlag)
-
-string(TOUPPER "${CMAKE_BUILD_TYPE}" U_CMAKE_BUILD_TYPE)
-
-include(set_compiler_flag.cmake)
-
-if(CPP17)
-  # User requested C++17, but compiler might not oblige.
-  set_compiler_flag(
-    _cxx_std_flag CXX
-    "-std=c++17"  # this should work with GNU, Intel, PGI
-    "/std:c++17"  # this should work with MSVC
-  )
-  if(_cxx_std_flag)
-    message(STATUS "Building with C++17")
-  endif()
-else()
-  set_compiler_flag(
-    _cxx_std_flag CXX REQUIRED
-    "-std=c++14"  # this should work with GNU, Intel, PGI
-    "/std:c++14"  # this should work with MSVC
-  )
-  message(STATUS "Building with C++14")
-endif()
-
-if(NOT _cxx_std_flag)
-  message(FATAL_ERROR "xtensor-blas needs a C++14-compliant compiler.")
-endif()
```

Comment:
Why did you need to change this logic? I think it was added to explicitely support "old" versions of CMake, is this change required?

Reply:
See now the separate PR https://github.com/xtensor-stack/xtensor-blas/pull/175